Client/Experiments/Experiments.swift

```diff
@@ -69,7 +69,7 @@ enum Experiments {
     }
 
     /// The `NimbusApi` object. This is the entry point to do anything with the Nimbus SDK on device.
-    public static let shared: NimbusApi = {
+    public static var shared: NimbusApi = {
```

Comment:
TIL: Swift considers `let` a signmal  that the `NimbusApi` is immutable. Because `globalUserParticipation` is a settable Swift considers it mutable, and requires a `var` here.

Reply:
The best way to fix this is to declare the `NimbusApi` as:

```swift
public protocol NimbusApi: class 
```

which tells Swift that  implementations have to be a class (and then you can use `let`.